bowtie/templates/report.html.j2

```diff
@@ -93,6 +102,15 @@
           {% endif %}
         </ul>
       </div>
+        <div id="theme-toggler" class="" style="margin-right: 1vw;">
+          <svg xmlns="http://www.w3.org/2000/svg" width="20" height="20" fill="currentColor" class="bi bi-sun" viewBox="0 0 16 16">
```

Comment:
I think this might need/want to be wrapped with a `button` element -- otherwise when you mouseover it the cursor doesn't change (from a pointer to a hand, say).

I noticed the same thing to be true about the `i` icons we added for runtime info, so if you manage to fix this here perhaps you can fix that one as well the same way.

Reply:
wrapped the icons (both **info** and **theme-toggler**) into `button` elements